lightning/src/blinded_path/message.rs

```diff
@@ -78,12 +79,62 @@ impl Writeable for ReceiveTlvs {
 	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), io::Error> {
 		// TODO: write padding
 		encode_tlv_stream!(writer, {
-			(6, self.path_id, option),
+			(65537, self.context, option),
 		});
 		Ok(())
 	}
 }
 
+impl Readable for ReceiveTlvs {
+	fn read<R: io::Read>(r: &mut R) -> Result<Self, DecodeError> {
+		_init_and_read_tlv_stream!(r, {
+			(65537, context, option),
+		});
+		Ok(ReceiveTlvs { context })
+	}
+}
+
```

Comment:
Is this needed? We are reading `ReceiveTlvs` indirectly when reading `ControlTlvs`.

Reply:
Just check that it is redundant!
Removed in [c8de6b6](https://github.com/lightningdevkit/rust-lightning/commit/c8de6b610fdde71730c5f536ecd89e9afcf003ea)
Thanks for the suggestion!